docs/api/tests.foreman.cli.rst

```diff
@@ -19,6 +19,20 @@
     :members:
     :undoc-members:
 
+:mod:`tests.foreman.cli.test_capsule`
+-------------------------------------
+
+.. automodule:: tests.foreman.cli.test_capsule
+    :members:
+    :undoc-members:
+
+:mod:`tests.foreman.cli.test_capsule_installer`
+-----------------------------------------------
+
+.. automodule:: tests.foreman.cli.test_capsule_installer
+    :members:
+    :undoc-members:
+
```

Comment:
As a follow-up pull request, we should make use of the `autodoc_default_flags` configuration option. See [here](https://github.com/SatelliteQE/nailgun/commit/b8fc67a9b75d807aae11185362ce21d02939fc57).